config.sample.yml

```diff
@@ -91,6 +91,13 @@ logging:
   # (Optional) Logging settings. You can have a severity debug,info,warn,error
   #
   level: info
+permissions:
+  # (Optional) Permissions for using the bridge
+  #
+  - actor: example.com
```

Comment:
is "actor" a (regex?) value for end user's mxid?

Reply:
I've hopefully documented this on the docs site now.